### DISCUSSION TITLE
Bug Report / Operational Change

### DISCUSSION THREAD

**Discussion by PedEchsing** [ViennaRSS/vienna-rss#1723] [General]
Vienna Version 3.9.0 :7ad9df044: (8175), MacOS Sonoma 14.2.1 (23C71):
With the latest version (3.9.0) lost the ability to open a site with Javascript disabled to read ONLY the text.
Example; Opening Aldous Pennyfarthing (snark) of Daily KOS to read the text only. Without the side-bars. Without any
videos from linked sites. without anything extraneous....

Please bring it back.
Kthxbye

**Comment by barijaona** [comment]
In fact, it's still there… your preference might have been missing in action during the transition.

Check menu item **Vienna** -> **Settings**, then the "_Advanced_" tab.

When I uncheck the "Enable Javascript in the internal browser", the page you mention appears as follows:


<img width="1238" alt="Capture d’écran 2024-01-22 à 10 46 54" src="https://github.com/ViennaRSS/vienna-rss/assets/442030/ac0c3b9d-b90b-48e3-aa00-c8598d1f7cbe">

**Reply by PedEchsing** [reply]
Yah... turning off (un-checking javascript) turns OFF all text...
checking js turns on everything.
Exactly what I don't want.
(By the way, you're looking at only column headlines. Not the associated text/articles. Which I get.)

**Comment by PedEchsing** [comment]
CLARIFICATION!!!
I've got 2 types of feeds for Daily KOS. 1 for Aldous Pennyfarthing ONLY and one for DK home page.
The crapout ONLY happens on the AP only page... when I open AP from the DK-home page, it does exactly what I want.
(I found that out after I left the first post...smh)

If I HAVE to, I can work around the problem by killing the Aldous-only feed and using the Home-page feed to get the reading experience I want. But that does also kill the convenience aspect. What can I say, I like AP, News Corpse, Bill in Portland, and the comics as separate feeds so I don't miss them in the daily feed.

SORRY IF THIS WAS NEEDED DATA TO FIND/FIX THE PROBLEM!

**Comment by PedEchsing** [comment]
OK!... one more time...
just got done checking. Aldous, News Corpse, Bill, etc. do NOT open properly, with javascript turned off, in a new tab.
They open, filled with grey boxes where the text OR graphics would be.
In both personal feeds or in D-KOS main feed.

**Comment by PedEchsing** [comment]
Well, if the text-feature only (when Javascript turned off) comes back, I'll be happy.
If it doesn't, I'll learn to live with it.

As it stands, I might as well close this discussion (clicking on 'close with comment' (?)),
but starting a new convenience discussion about right-clicks.
